pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -1058,20 +1065,64 @@ public static int[] millisecondMV(long[] millis, String timezoneId) {
     return results;
   }
 
+
   /**
    * The sql compatible date_trunc function for epoch time.
-   *
-   * @param unit truncate to unit (millisecond, second, minute, hour, day, week, month, quarter, year)
-   * @param timeValue value to truncate
-   * @return truncated timeValue in TimeUnit.MILLISECONDS
    */
   @ScalarFunction(names = {"dateTrunc", "date_trunc"})
+  public static class dateTruncScalarFunctions {
```

Comment:
changes that needs to be made to all `ScalarFunction` with `TransformFunctionType` associated with